rootfs/rootfs/etc/rc.d/automount-shares

```diff
@@ -18,7 +18,7 @@ if modprobe vboxguest &> /dev/null && modprobe vboxsf &> /dev/null; then
 		name="${2:-$dir}"
 		
 		mkdir -p "$dir" 2>/dev/null
-		if ! mount -t vboxsf -o "$mountOptions" "$name" "$dir" 2>/dev/null; then
+		if ! mount.vboxsf -o "$mountOptions" "$name" "$dir" 2>/dev/null; then
```

Comment:
Why this change too?  `mount -t xyz` is supposed to shell out to `mount.xyz` appropriately as necessary by itself.